kernel/Kconfig

```diff
@@ -1040,6 +1040,18 @@ config KERNEL_COHERENCE
 	  may fail strangely.  Some assertions exist to catch these
 	  mistakes, but not all circumstances can be tested.
 
+config TICKET_SPINLOCKS
+	bool "Ticket lock spinlocks providing lock acquisition fairness"
```

Comment:
```suggestion
config TICKET_SPINLOCKS
	bool "Ticket spinlocks for lock acquisition fairness [EXPERIMENTAL]"
	select EXPERIMENTAL
```

Reply:
Hi @cfriedt ,

Sure, let's mark it as experimental.
Thanks!